### DISCUSSION TITLE
Executing LDT error from LISF-x86_64.AppImage with testcase data

### DISCUSSION THREAD

**Discussion by smf1959-mor** [NASA-LIS/LISF#1496] [Q&A] [answered]
I recently downloaded the **LISF_7.5.0 public release** image **LISF_x86_64.AppImage** and the  data  **lis-public-testcases** . when I attempt to run LDT by the following command **$HOME/LIS_F_image/LISF-x86_64.AppImage -a LDT ldt.config.noah36_params** in the folder contening lis-public-testcases I have the following error **/tmp/.mount_LISF-x8c4mn3/usr/bin/LDT: error while loading shared libraries: libmpifort.so.12: cannot open shared object file: No such file or directory**.
Could you indicate me what is wrong. Please note also that I m new in LIS Framework.

**Comment by jvgeiger** [comment, marked as answer]
Hello,

The error message means that the AppImage container cannot find the MPI libraries. The AppImage container requires that the MPI libraries to be installed on your computer.  So, either the MPI libraries are not installed or your installation needs some additional symbolic links.

Would you please verify that MPI is installed?


Thanks,
Jim
